volume-providers/softlayer/block/block_volume_manager.go

```diff
@@ -258,11 +258,11 @@ func (sls *SLBlockSession) VolumeCreateFromSnapshot(snapshot provider.Snapshot,
 	originalStorageType := *originalVolume.StorageType.KeyName
 	isPerformanceVolume := false
 	finalPrices := []datatypes.Product_Item_Price{}
+	volumeCategory := fmt.Sprintf(`storage_%s`, "block")
```

Comment:
Can you please check `volumeCategory` for endurance and performance are different, Can you please verify for both endurance and performance storage

Reply:
I tested code for both performace and endurance